spec/requests/device_story_comments_controller_spec.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe DeviceStoryCommentsController, type: :request do
+  let!(:device_story) { Fabricate(:device_story, last_location_name: 'Israel', device_urn: 'safecast:114699387') }
+  let!(:user) { Fabricate(:user) }
+  let!(:device_story_comment) { Fabricate(:device_story_comment) }
+  describe 'POST #create' do
+    WebMock.allow_net_connect!
```

Comment:
I'd take this out and use the calls in https://github.com/joshfrench/rakismet#testing to ensure rakismet doesn't actually try to check spam during testing.

Reply:
I would use WebMock to check if it sends request to Akismet.

For example:

```ruby
let(:addressable) { Addressable::Template.new('https://{key}.rest.akismet.com/1.1/comment-check') }
...
before { stub_request(:post, addressable) }
...
  expect(WebMock).to have_requested(:post, addressable)
```